components/dashboards-web-component/src/index.js

```diff
@@ -0,0 +1,143 @@
+import GoldenLayout from 'golden-layout';
+import 'golden-layout/src/css/goldenlayout-base.css';
+import 'golden-layout/src/css/goldenlayout-light-theme.css';
+import BarChart from './BarChart';
+import PieChart from './PieChart';
+import Publisher from './Publisher';
+import Subscriber from './Subscriber';
+import DynamicWidget from './DynamicWidget';
+import EmptyWidget from './EmptyWidget';
```

Comment:
Let's load all the widgets from the dynamic loading approach without using direct imports